### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by generall** [qdrant/qdrant#469] [General]
# Change log

## :carousel_horse: Features

* Qdrant now supports **any JSON as payload** - https://github.com/qdrant/qdrant/pull/306
  * The uploaded payload structure now exactly matches the retrieved results.
  * The payload schema is now obsolete. The payload type is now used for field indexes only.

* Alias API in gRPC - Making up for a lost API https://github.com/qdrant/qdrant/pull/408

* New filtering condition
    * Bool filter and refactoring of `Match` - https://github.com/qdrant/qdrant/pull/421
    * `IsEmpty` filter - https://github.com/qdrant/qdrant/pull/423
    * ValuesCount filter - https://github.com/qdrant/qdrant/pull/439

## :rocket: Improvements

* Geo payload indexing - https://github.com/qdrant/qdrant/pull/366
  * Geolocation queries are now processed with a special type of index.

* Removed BLAS - https://github.com/qdrant/qdrant/pull/182
    * Custom, slim, architecture-aware implementations now perform distance computation.
    * Additional support for neon aarch64.
    * Euclidean metric now uses SIMD - order of magnitude faster performance.

* Use Filtering context instead of Condition Checker - https://github.com/qdrant/qdrant/pull/413
  * Opens up opportunities for additional optimization during the planning phase of the queries.

* `sled` removed - https://github.com/qdrant/qdrant/pull/402
  * Less overhead resources.
  
* HNSW Performance improvements
    * Better iteration implementation - https://github.com/qdrant/qdrant/pull/409
    * Chunked Vector storage - https://github.com/qdrant/qdrant/pull/457

## :heavy_exclamation_mark: Breaking changes

* Due to significant changes in payload format, collections created with the previous engine version are not compatible with the new version. The most straightforward migration approach would be to re-create collections with the latest version from scratch. Breaking changes are required to make our v1.0 release as legacy-free as possible. Hope for your understanding.

## :microscope: Work-in-progress

* Our main efforts are now focused on the distributed deployment feature. Stay tuned a check out our [Roadmap](https://github.com/qdrant/qdrant/blob/master/docs/roadmap/README.md) for details.

<hr /><em>This discussion was created from the release <a href='https://github.com/qdrant/qdrant/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by orsk-moscow** [comment]
> Breaking changes are required to make our v1.0

so, there won't any versions with broken backward compatibility, in foreseeable future at least?
